CODEOWNERS

```
@@ -225,7 +225,7 @@
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
-* @cilium/tophat
+* @cilium/committers
```

I realized that the other common pattern for PRs on stable branches is when preparing a release. We may want to consider adding them in as well as optional approvers:

```suggestion
* @cilium/committers @cilium/release-managers
```

---

Let's see how things work out without `@cilium/release-managers`